bom/runtime/pom.xml

```diff
@@ -867,6 +867,35 @@
       <version>0.5.8</version>
     </dependency>
 
+    <!-- JAXB -->
+    <dependency>
```

Comment:
It looks like we now included some of these dependency twice:

https://github.com/openhab/openhab-core/blob/b6aae6907f3d247e5d590bb25f6cb009e17584f8/bom/runtime/pom.xml#L135-L141

https://github.com/openhab/openhab-core/blob/b6aae6907f3d247e5d590bb25f6cb009e17584f8/bom/runtime/pom.xml#L127-L133

https://github.com/openhab/openhab-core/blob/b6aae6907f3d247e5d590bb25f6cb009e17584f8/bom/runtime/pom.xml#L159-L165